.env

```diff
@@ -10,6 +10,18 @@ ACP_URL=https://localhost:8443
 # if you deploy acp on a separate host, set this env variable to the same value as ACP_URL
 ACP_INTERNAL_URL=https://acp:8443
 
+# url to developer third party provider portal
+TPP_URL=https://localhost:8090
+
+# url to financroo third party provider portal
+FINANCROO_URL=https://localhost:8091
+
+# url to consent self service portal
+CONSENT_SELF_SERVICE_URL=https://localhost:8085
+
+# url to consent admin portal
+CONSENT_ADMIN_URL=https://localhost:8086
```

Comment:
Since those env variables are related to tests only I would put them in a separate .env file for tests or at least add a prefix.

Reply:
I've moved those variables to cypress.json file.